region/wreckedship/main.json

```diff
@@ -2014,8 +2046,25 @@
                     "Morph",
                     "f_DefeatedPhantoon"
                   ],
-                  "note": "The path can't be passed unless the Workrobot is activated (so Phantoon dead)",
-                  "devNote": "This doesn't require breaking the bomb blocks on the premise that you can't here without breaking them."
+                  "note": "By normal means, the path can't be passed unless the Workrobot is activated (so Phantoon dead)",
+                  "devNote": "This doesn't require breaking the bomb blocks on the premise that you can't get here without breaking them."
+                },
+                {
+                  "name": "WS East Supers Crystal Flash Clip (Right to Left)",
+                  "notable": true,
+                  "requires": [
+                    "canCrystalFlash",
+                    "canCeilingClip"
+                  ],
+                  "note": [
+                    "Crystal flash to force a standup then spin jump up then morph to bypass the dead robot. Use Coverns to damage down if necessary.",
```

Comment:
I was aim down jumping and just falling out before reading this.  No reason to add unnecessary alternatives though.